app_dev.yaml

```diff
@@ -150,6 +150,17 @@ handlers:
     X-Xss-Protection: "1; mode=block"
   secure: always
   expiration: "0"
+- url: /account-to-be-deleted
```

Comment:
Should we be hiding this page (or at least POST requests from it) behind a boolean flag for now?